test-image/Dockerfile

```
@@ -1,6 +1,8 @@
 FROM node:10-slim
-ENV NODE_PORT 3000
-COPY package* ./
-RUN npm install
-COPY . .
-CMD ["npm", "start"]
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+COPY index.js .
+ADD ./config "/config"
```

how about be consistent with copy here too?
```suggestion
COPY config/ /config
```

---

also config probably changes less frequently than the code, this is a test app and all but I would either.
1. move CONFIG above index.js
2. combine into one layer
3. just move back to COPY . . and make sure the dockerignore is set right
4. not bother because this is a tiny test app

---

- ADD was used so the whole folder is included including 1 or more config files.
- I used the config because the PORT was configured so adding this to the default.yaml is sensible for testing.